app/views/users/show.html.erb

```diff
@@ -1,14 +1,13 @@
 <main>
+  <%= render partial: 'shared/user_card', locals: {user: @user} %>
   <article>
-    <h1><%= @user.name %></h1>
-    <p>Posts: <%= @user.posts_counter %></p>
     <p>Bio: <%= @user.bio %></p>
-  </article>
-
+<div class="post-user">
+    <small><%= @user.name %>'s posts </small>
   <% @user.posts.each do |post| %>
 
     <%= link_to user_posts_path(@user) do %>
-      <h2><%= post.title %></h2>
+      <h2 class="post-title"><%= post.title %></h2>
```

Comment:
- [ ] As stated in the requirement and the wireframe, kindly ensure that only the `first three recent posts` for a user are displayed; don't forget to implement a `see all link` that will redirect to the `user_posts_path `to display all the posts. 👍 

- [ ] Kindly add the `likes button` to the `user _show`, `post_index` and `post_show` pages as given in the template. 👍 

<img width="244" alt="Untitled" src="https://github.com/naji-97/rails-blog/assets/103386183/4d9931a7-209c-4504-8073-317feb19c30a">